Katello/hammer-cli-katello
# Fixes #36970 - Add ansible collections and other missing content counts to content counts

Adds more content types to `hammer repository info`.

Also ensures that blank content is hidden in `hammer capsule content info`

To test:

1) Test `hammer repository info` and check that content counts are listed for all content types
2) Quickly check that `hammer capsule content info` still shows counts correctly
